libs/application/templates/passport/src/lib/PassportTemplate.ts

```diff
@@ -92,7 +121,7 @@ const PassportTemplate: ApplicationTemplate<
     },
   },
   mapUserToRole(_id: string, _application: Application): ApplicationRole {
-    return 'applicant'
+    return Roles.APPLICANT
```

Comment:
Can you check the applicant id and return the role and return undefined if it doesnt match the user?
```
mapUserToRole(
    nationalRegistryIdOfAuthenticatedUser: string,
    application: Application,
  ): ApplicationRole | undefined {
    if (nationalRegistryIdOfAuthenticatedUser === application.applicant) {
      return Roles.APPLICANT
    }
    return undefined
  },
```